script.js

```diff
@@ -5,24 +5,75 @@ var game = {
   players: [],
   hands: [],
   buildDeck: function(){
-
+    var numberOfCardValues = values.length;
+    var numberSuits = suits.length;
+    var numberOfCards = 1;
+    for(var v = 0; v < numberOfCardValues; v++){
+      for(var s = 0; s < numberSuits; s++){
+        var card = values[v]+"-"+suits[s];
+        game.deck.push(card);
+        console.log(game.deck.length);
+      }
+    }return game.deck;
   },
   shuffleDeck: function(){
+    //Shuffle ARRAY Elements
+          var currentIndex = game.deck.length;
+          var temporaryValue;
+          var randomIndex;
+
+          // While there remain elements to shuffle...
+          while (0 !== currentIndex) {
```

Comment:
this works! typically for loops are used when you know how many times to loop. in this case `game.deck.length`